src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// Provides support for computing a hash or HMAC value incrementally across several segments.
+    /// </summary>
+    public sealed class IncrementalHash : IDisposable
```

Comment:
Ah :smile:.  Since MSDN doesn't know about IncrementalHash we probably left it as the first nuget-only type in this package/assembly.  So the partial facade for desktop would have needed an implementation, and I didn't want to get bitten by something straddling the internal/open sources.

Hopefully this file could be removed, and the IncrementalHash implementation already in this library would work in the desktop partial facade DLL.